web/src/components/bgImagewithCard.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BackgroundImage from 'gatsby-background-image';
+
+import Card from './card';
+
+export default function BgImagewithCard({ image }) {
+    const {
+        childImageSharp: { fluid },
+    } = image;
+
+    return (
+        <BackgroundImage
+            Tag="section"
+            className="w-full min-h-screen"
```

Comment:
Can you use flex to position the card element like you did with the hero component? Then you could remove the positioning classes from the card element.